server/rpm/pbench-server.spec.j2

```diff
@@ -136,6 +136,8 @@ fi
 /%{installdir}/lib/pbench/server/api/resources/query_apis/__init__.py
 /%{installdir}/lib/pbench/server/api/resources/query_apis/elasticsearch_api.py
 /%{installdir}/lib/pbench/server/api/resources/query_apis/query_controllers.py
+/%{installdir}/lib/pbench/server/api/resources/query_apis/query_results.py
+/%{installdir}/lib/pbench/server/api/resources/query_apis/query_result.py
```

Comment:
I thought I heard Peter describing how we could specify a directory in this list instead of individual files...is that something that we should be pursuing here (if not wholesale, then piecewise covering changes)?

Reply:
And ... we definitely should do it, but I'd rather not add that basic change into one of these PRs. Someone (hint, hint) could do that independently, and the results could be merged into these ...

Reply:
I was just wondering whether, instead of adding two files here, you could just add their directory?  (Less change and all that.... ;-) )